### PR TITLE
Finish Symfony 5 support.

### PR DESCRIPTION
**Pull request by dereuromark**
Start to resolve 
- https://github.com/propelorm/Propel2/issues/1574
- https://github.com/propelorm/Propel2/issues/1573
- https://github.com/propelorm/Propel2/issues/1578

by
- adding Symfony5
- dropping EOL versions to simplify the very broad dependency spectrum
- using proper semver on this low level

Now only PHPUnit is outdated and needs in the future to go to 8+

This is what we should be supporting moving forward
Now we just need to start fixing the issues showing up.

EDIT: I will rebase mine on top of https://github.com/propelorm/Propel2/pull/1595 now.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
         "psr/log": "^1.0"
     },
     "require-dev": {
+        "symfony/translation": "^3.4.0||^4.3.0||^5.0.0",
```
Comment (Incognito) [inline composer.json:30]: it's a hard requirement of ValidateBehavior.php.

Comment (dereuromark) [reply composer.json:30]: Did you add it to your changes for require so far?

Comment (dereuromark) [reply composer.json:30]: Ah I see, I will revert this then.

File: src/Propel/Generator/Command/TestPrepareCommand.php [hunk 2]
```diff
@@ -95,25 +95,30 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
+        $result = static::CODE_SUCCESS;
         foreach ($this->fixtures as $fixturesDir => $connections) {
-            $this->buildFixtures(sprintf('%s/%s', self::FIXTURES_DIR, $fixturesDir), $connections, $input, $output);
+            $run = $this->buildFixtures(sprintf('%s/%s', self::FIXTURES_DIR, $fixturesDir), $connections, $input, $output);
+            if ($run !== static::CODE_SUCCESS) {
+                $result = $run;
+            }
```
Comment (Incognito) [inline src/Propel/Generator/Command/TestPrepareCommand.php:103]: I really like this "un-masking" of error codes.

Comment (dereuromark) [reply src/Propel/Generator/Command/TestPrepareCommand.php:103]: so far the return value was unused, yeah :)

**Pull request merged**